lib/row_transformer.rb

```diff
@@ -39,9 +39,9 @@ def transform
     end
 
     def load_simple_fields
-        @formatted_row.box_id = @transformed_row['id']
-        @formatted_row.holding_id = @transformed_row['record_num']
-        @formatted_row.box_count = @transformed_row['box_count']
+        @formatted_row.box_id = @transformed_row['id'].to_i
+        @formatted_row.holding_id = @transformed_row['record_num'].to_i
+        @formatted_row.box_count = @transformed_row['box_count'].to_i
         @formatted_row.claim_count = @transformed_row['claim_cnt']
         @formatted_row.copy_count = @transformed_row['copy_cnt']
```

Comment:
Should this cover `claim_cnt` and `copy_cnt` as well?

Reply:
Good point! I missed these because they are allowed to be null, but we don't want them to be strings.

Reply:
looks great, just gotta fix that linting error then this should be good to go!